index.html

```diff
@@ -64,8 +64,11 @@
       <br>
       <a href="#" class="[ button ][ ffab before remove fa-arrow-left ]">Transition Remove</a>
       <a href="#" class="[ button ][ ffab remove fa-arrow-right ]">Transition Remove</a>
+      <hr>
+      <strong>Hover/Active State</strong>
       <br>
-      <a href="#" class="[ button ][ ffab static fa-arrow-right ]">No Transition (Static)</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right ]">With Hover/Active</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right no-hover-active ]">Without Hover/Active</a>
```

Comment:
If there is a way to turn on/off the hover effect and the active state individually, I think it would be useful.
Also it seems like there might be a way to use the sass setting to determine whether or not to include it, without have to add a class.  Is that possible?